pybrake/test_celery_integration.py

```diff
@@ -21,6 +21,27 @@ def do_POST(self):
         self.wfile.write('{"id":"1"}'.encode("utf-8"))
 
 
+def test_celery_integration_patched():
+    server_address = ("", 8080)
+    server = HTTPServer(server_address, Handler)
+
+    httpd_thread = threading.Thread(target=server.serve_forever)
+    httpd_thread.daemon = True
+    httpd_thread.start()
+
+    raise_error_patched.apply()
+
+    for _ in range(10):
+        if notice is None:
+            time.sleep(1)
+        else:
+            break
+
+    assert notice is None
+
+    server.socket.close()
```

Comment:
Will `server` run forever if the assert on line 40 fails?

Reply:
Yes, until and unless all test cases are finished.

Reply:
As a result, I'm shutting down this test case before it finishes, and there are no other cases that are dependent on it.

Reply:
What do you you mean by "shutting down" the test? Is there a way to refactor the tests so that they don't depend on each other?

Reply:
My apologies. It's not shutting down the test, but it is shutting down this server on line 37.